### PR TITLE
2.2.5 updates

### PR DESCRIPTION
**Pull request by arvindkrishnakumar-okta**

### CODE REVIEW

File: resources/spec.yaml [hunk 1]
```diff
@@ -10,7 +10,7 @@ info:
   license:
     name: Apache-2.0
     url: 'http://www.apache.org/licenses/LICENSE-2.0.html'
-  version: 1.12.0
```
Comment (arvindkrishnakumar-okta) [inline resources/spec.yaml:13]: Not sure why this is so out of date.

**Review by laura-rodriguez — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,11 @@
 # Changelog
-Running changelog of releases since `2.2.3`
+Running changelog of releases since `2.2.4`
```
Comment (bretterer) [inline CHANGELOG.md:2]: Why was this updated to 2.2.4?

File: resources/spec.yaml [hunk 3]
```diff
@@ -10,7 +10,7 @@ info:
...
+  version: 2.2.5
```
Comment (bretterer) [inline resources/spec.yaml:13]: Please remove this version from the spec.  It is automatically added during build

File: resources/spec.yaml [hunk 4]
```diff
@@ -10,7 +10,6 @@ info:
   license:
     name: Apache-2.0
     url: 'http://www.apache.org/licenses/LICENSE-2.0.html'
-  version: 1.12.0
```
Comment (arvindkrishnakumar-okta) [inline resources/spec.yaml:13]: Removed this per your remark @bretterer.

File: resources/spec.yaml (hunk 3 above)
Comment (arvindkrishnakumar-okta) [reply resources/spec.yaml]: done

File: CHANGELOG.md (hunk 2 above)
Comment (arvindkrishnakumar-okta) [reply CHANGELOG.md:3]: fixed

**Pull request merged**